Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,31 +1,34 @@
 # L5: Principal Software Engineer
 
-> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
+> _I am an influential Engineering leader who identifies problems and opportunities relevant to Octopus product and technical strategies, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus._
 
-- **Planning horizon**: 1+ year
-- **Impact radius**: Group+ (15-50). Note: This role is evolving as the company grows. PSE is right at the top of our IC track. For example, the impact radius expectations of an L5 will likely grow as Octopus grows larger over the next couple years, from 2-4 teams in 2021. We expect the role to stabilise in 2022 once the company has grown to the point of needing an L6 IC engineering role.
-- **Evaluation**: Director or VP
-- **Responsibility and direction needed**: Drives technical strategy in alignment with product strategy and drives high-impact improvements across teams.
+- **Thinking horizon**: 1 year
+- **Impact radius**: Group
+- **Evaluation**: Director
+- **Responsibility and direction needed**: I drive technical strategy in alignment with product strategy for my group. Point me in the right direction with helpful constraints and good outcomes happen with minimal oversight.
 
 ## 🦉 Domain expertise
 
-- I contribute to core technologies used by a variety of teams.
-- I help teams make critical product architecture and implementation decisions.
-- I maintain a vision of the overall technical direction within my impact radius.
-- I contribute to the engineering standards and principles of the company.
-- I contribute to the Octopus technical strategy.
-- I manage technical risks, and architect to reduce technical risk.
+- I maintain a clear and understandable vision for the technical direction for teams within my group.
```

Comment:
I'd like to understand this point further. Do we mean an explicit vision for the technical direction each team should be following? We talk about target achitecture below, but I suspect we are more talking about helping teams set and follow technical direction within their missions.

Reply:
For context: This isn't new. I moved this existing point from another section. But I'm glad you're asking for clarity.

Yes, I see two distinct (but sometimes connected) traits:

- contribution to + championship of the target architecture
- being accountable for the technical direction their specific team(s) are taking to deliver specific outcomes

In some cases, these may intersect or link, but in others, they won't.

For Directors, I often say:

- Your first responsibility is to your group - are they winning?
- If your group is winning, work on something broader-scoped

The same guidance applies to PSEs. What do you think?